build.cake

```diff
@@ -554,41 +526,26 @@ Task("DownloadNdk")
     }
 }).OnError(HandleError);
 
-void GetUwpPackage (AppCenterModule module, bool usePublicFeed) {
+async Task GetUwpPackage (AppCenterModule module, bool usePublicFeed) {
     // Prepare destination
     var destination = "Assets/AppCenter/Plugins/WSA/" + module.Moniker + "/";
     EnsureDirectoryExists (destination);
     DeleteFiles (destination + "*.dll");
     DeleteFiles (destination + "*.winmd");
     var nupkgPath = "externals/uwp/";
+    Information($"GetUwpPackage use public feed {usePublicFeed}");
```

Comment:
Move this inside if to use more accurate wording. Also, please avoid usage internal method names in logs